README.md

```diff
@@ -16,6 +16,7 @@ Pull requests or issues welcome to add more. I surely forgot a bunch.
 
 # Behaviors
 
+<a href="#me-too-plus-one"></a>
 ## Me too, plus one
```

Comment:
I can't find concrete docs that this is supported by GitHub-flavored Markdown.

How about we just go with HTML instead, ala https://gist.github.com/asabaylus/3071099?permalink_comment_id=4585378#gistcomment-4585378?

So `<h2 id='me-too-plus-one'>Me too, plus one</h2>`

Then want to add a little TOC to the top of the file using the links as an example of it working?

Reply:
It is absolutely supported because it’s regular html, and i use it in hundreds of repos.